solara/components/file_drop.vue

```diff
@@ -1,6 +1,13 @@
 <template>
   <div ref="dropzone" class="solara-file-drop" effectAllowed="move">
-    {{ (file_info && file_info.length === 1) ? file_info[0].name : label }}
+    <div v-if="file_info && file_info.length">
+      <ul>
+        <li v-for="file in file_info">{{ file.name }}  {{ file.size }} bytes</li>
```

Comment:
I would like the list to not have bullet points.

Reply:
Done!